shared/fs/folder/offline.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as Kb from '../../common-adapters'
+import React from 'react'
+import * as Styles from '../../styles/index'
+import * as Types from '../../constants/types/fs'
+import TopBar from '../top-bar'
+
+type Props = {|
+  path: Types.Path,
+|}
+
+const OfflineFolder = (props: Props) => (
+  <Kb.Box2 direction="vertical" style={styles.contentContainer} fullWidth={true} alignItems={'stretch'}>
+    <Kb.BoxGrow>
+      <TopBar path={props.path} mode={'offline'} />
+      <Kb.Box2
+        direction="vertical"
+        style={styles.emptyContainer}
+        fullHeight={true}
```

Comment:
For the same reason as before, this can't be `fullHeight`. Otherwise it'd try to push the `TopBar`. Just use `flex: 1`.